exchanges/kraken.py

```diff
@@ -9,6 +15,20 @@ def __init__(self):
         self.timeout = 5.0
         self.url = 'https://api.kraken.com/0/'
 
+        # Load the API keys from the data folder.
+        key_file = os.path.join(PRIVATE_DATA_DIR, 'kraken-key')
+        if os.path.exists(key_file):
+            with open(key_file, 'rt') as f:
+                self.public_key = f.readline().rstrip()
+                self.private_key = f.readline().rstrip()
+        else:
+            raise Warning('The Kraken API requires an API key to access '
```

Comment:
I don't think this warning is necessary. I can think of situations where I only want to make public queries to the server, for example, when I want to compare the price of a currency across exchanges.